src/page-templates/main-content-page/content-page-data.ts

```diff
@@ -0,0 +1,75 @@
+import { ContentItem } from '../../interfaces/content-item';
+import { pillCategoryToSlug } from '../../types/pill-category';
+import { getSideNav } from '../../service/get-side-nav';
+import { setURLPathForNavItems } from '../../utils/format-url-path';
+import { getMetaInfoForTopic } from '../../service/get-meta-info-for-topic';
+import { getRelatedContent } from '../../utils/get-related-content';
+import { getBreadcrumbsFromSlug } from '../../components/breadcrumbs/utils';
+import { Crumb } from '../../components/breadcrumbs/types';
+import { appendDocumentationLinkToSideNav } from '../../utils/add-documentation-link-to-side-nav';
+import { determineVideoOrPodcast } from './content-page-template';
+import { getContentItemFromSlug } from '../../service/get-content-by-slug';
+
+let pluralize = require('pluralize');
+
+export const getContentPageData = async (slug: string[]) => {
+    const slugStr = slug.join('/');
+    const contentItem: ContentItem | null = await getContentItemFromSlug(
```

Comment:
This will fetch only a single piece of content, which was previously using `getAllContentItems`.